integrations/terraform-cloud/.port/resources/blueprints.json

```diff
@@ -1,4 +1,64 @@
 [
+  {
+    "identifier": "terraformCloudOrganization",
+    "description": "This blueprint represents an organization in Terraform Cloud",
+    "title": "Terraform Cloud Organization",
+    "icon": "Terraform",
+    "schema": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "title": "Organization Name",
+          "description": "The name of the Terraform Cloud organization"
+        },
+        "externalId": {
+          "type": "string",
+          "title": "External ID",
+          "description": "The external ID of the organization"
+        },
+        "createdAt": {
+          "type": "string",
+          "format": "date-time",
+          "title": "Created At",
+          "description": "The time at which the organization was created"
+        },
+        "email": {
+          "type": "string",
+          "title": "Email",
+          "description": "The email associated with the organization"
+        },
+        "collaboratorAuthPolicy": {
+          "type": "string",
+          "title": "Collaborator Authentication Policy",
+          "description": "Policy for collaborator authentication"
+        },
+        "planIdentifier": {
+          "type": "string",
+          "title": "Plan Identifier",
+          "description": "The identifier of the plan the organization is on"
+        },
```

Comment:
I think we don't need that
add the following instead:
        "plan-expired": false,
        "plan-expires-at": null,